paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/HeaderTextFactory.kt

```diff
@@ -16,55 +16,72 @@ internal class HeaderTextFactory(
         types: List<PaymentMethodCode>,
     ): Int? {
         return if (isCompleteFlow) {
-            when (screen) {
-                is PaymentSheetScreen.SelectSavedPaymentMethods -> {
-                    if (isWalletEnabled) {
-                        null
-                    } else {
-                        R.string.stripe_paymentsheet_select_payment_method
-                    }
-                }
-                is PaymentSheetScreen.AddFirstPaymentMethod, PaymentSheetScreen.VerticalMode -> {
-                    R.string.stripe_paymentsheet_add_payment_method_title.takeUnless {
-                        isWalletEnabled
-                    }
-                }
-                is PaymentSheetScreen.EditPaymentMethod -> {
-                    StripeR.string.stripe_title_update_card
-                }
-                is PaymentSheetScreen.Loading,
-                is PaymentSheetScreen.AddAnotherPaymentMethod,
-                is PaymentSheetScreen.Form,
-                null -> {
-                    null
-                }
-            }
+            createForCompleteFlow(screen, isWalletEnabled)
```

Comment:
ran into a linter issue about this function being too complex, so separated some of it out

Reply:
The only actually new thing in this file is the PaymentSheetScreen.ManageSavedPaymentMethods branch